### PR TITLE
Include 'source_dir' in cobertura report

### PR DESCRIPTION
**Pull request by svenfoo**
If a source_dir is specified on the command-line, then all files in the
cobertura report are specified relative to that folder. The report should
then mention that source_dir instead of '.' so that files are found for
coverage display.

### CODE REVIEW

File: src/cobertura.rs [hunk 1]
```diff
@@ -232,14 +232,13 @@ impl ToString for ConditionType {
...
     results: CovResultIter,
+    sources: Vec<String>,
     demangle: bool,
     demangle_options: DemangleOptions,
 ) -> Coverage {
-    let sources = vec![".".to_owned()];
     let packages: Vec<Package> = results
         .map(|(_, rel_path, result)| {
             let all_lines: Vec<u32> = result.lines.iter().map(|(k, _)| k).cloned().collect();
-
```
Comment (marco-c) [inline src/cobertura.rs:242]: Nit: please reinstate the newline, as it is an unrelated change

**Review by marco-c — APPROVED** [review]
Thanks!

File: src/cobertura.rs (hunk 1 above)
Comment (svenfoo) [reply src/cobertura.rs]: No problem. Done.

**Review by marco-c — APPROVED** [review]
